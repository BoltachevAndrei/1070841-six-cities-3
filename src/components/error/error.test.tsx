import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
import history from '../../history';
import Error from './error';

it(`Error renders correctly`, () => {
  const tree = renderer
    .create(
        <Router history={history}>
          <Error />
        </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
