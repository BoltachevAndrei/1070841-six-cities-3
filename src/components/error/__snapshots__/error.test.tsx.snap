// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="/img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <a
                className="header__nav-link header__nav-link--profile"
                href="/login"
                onClick={[Function]}
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                />
                <span
                  className="header__user-name user__name"
                >
                  Sign In
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>,
  <main
    className="page__main"
  >
    <div
      className="container"
    >
      <section
        className="error"
      >
        <h1
          className="visually-hidden"
        >
          Error
        </h1>
        <div
          style={
            Object {
              "marginLeft": "auto",
              "marginRight": "auto",
              "marginTop": "16.7vh",
              "paddingTop": "94px",
              "textAlign": "center",
              "width": "420px",
            }
          }
        >
          <b
            className="favorites__status"
          >
            Error
          </b>
          <p
            className="favorites__status-description"
          >
            Something went wrong.
          </p>
          <p
            className="favorites__status-description"
          >
            Please try again later.
          </p>
        </div>
      </section>
    </div>
  </main>,
]
`;
